### ISSUE TITLE
Cannot tab through Header components links/buttons

### ISSUE THREAD

**Issue by ibdf** [CityOfPhiladelphia/phila-ui#89]
Header needs to be updated to allow tabbing through the navigation links (top right).

**Comment by ibdf** [comment]
You can actually tab through them but you can't tell because the focus color is not added to the buttons. We might remove buttons altogether from the header.
